packages/flutter_tools/test/base/logger_test.dart

```diff
@@ -281,11 +281,9 @@ void main() {
     testUsingContext('sequential startProgress calls with StdoutLogger', () async {
       context[Logger].startProgress('AAA')..stop();
       context[Logger].startProgress('BBB')..stop();
-      expect(outputStdout(), <String>[
-        'AAA                                                               0ms',
-        'BBB                                                               0ms',
-        '',
-      ]);
+      expect(outputStdout().length, equals(3));
+      expect(outputStdout()[0], matches(RegExp(r'AAA[ ]{63}[\d]+ms')));
```

Comment:
Will it still be 63 spaces if it's 10ms?

Reply:
No.  Fixed to take up to four digits or spaces at the end.  If it takes more than 10 seconds to do start..stop, then there's something else afoot, and it should fail.